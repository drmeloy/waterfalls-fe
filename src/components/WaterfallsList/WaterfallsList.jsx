import React, { useState } from 'react';

export default function WaterfallsList(){
  const [waterfalls, setWaterfalls] = useState([]);
  
  

  render (

  );
}
