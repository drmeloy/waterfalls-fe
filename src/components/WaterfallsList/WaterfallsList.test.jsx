import React from 'react';
import { shallow } from 'enzyme';
import WaterfallsList from './WaterfallsList';

describe('WaterfallsList component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<WaterfallsList />);
    expect(wrapper).toMatchSnapshot();
  });
});
