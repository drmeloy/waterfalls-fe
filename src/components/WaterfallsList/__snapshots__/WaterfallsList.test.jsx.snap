// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WaterfallsList component matches snapshot 1`] = `
<Fragment>
  <div>
    MAP GOES HERE
  </div>
  <div
    className="waterfalls--list"
  >
    <table
      className="table"
    >
      <thead
        key="thead"
      >
        <tr>
          <th>
            Name
          </th>
          <th>
            Height
          </th>
          <th>
            Longitude
          </th>
          <th>
            Latitude
          </th>
          <th>
            Description
          </th>
          <th>
            Actions
          </th>
        </tr>
      </thead>
      <tbody />
    </table>
  </div>
</Fragment>
`;
